greenkeeper.json

```diff
@@ -33,5 +33,6 @@
         "sandbox/example/package.json"
       ]
     }
-  }
+  },
+  "ignore": ["@types/node"]
```

Comment:
> chore: ignore @types/node upgrade by greenkeeper

What is the reasoning for this exclusion? Can you move this commit to a new pull request please? I don't see how it's related to RestServer improvements.

Reply:
Sorry, I guess I accidentally created this branch from the wrong base.